lib/v2/xmut/jwc/bkjw.js

```diff
@@ -0,0 +1,64 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+const xmut = 'https://jwc.xmut.edu.cn';
+
+module.exports = async (ctx) => {
+    const { category = 'jwxt' } = ctx.params;
+    const url = `${xmut}/index/tzgg/${category}.htm`;
+    const res = await got(url, {
+        headers: {
+            referer: xmut,
+        },
+        https: {
+            rejectUnauthorized: false,
+        },
+    });
+
+    const $ = cheerio.load(res.data);
+    const itemsArray = [];
+
+    $('#result_list table tbody tr').each((index, row) => {
+        const res = $('td', row).eq(0);
+        const resDate = $('td', row).eq(1);
+        const resLink = $('a', res).attr('href');
+        let link;
+
+        if (resLink.startsWith('../../')) {
+            link = `${xmut}/${resLink}`;
+        } else {
+            link = resLink;
+        }
```

Comment:
https://github.com/DIYgod/RSSHub/pull/13326#issuecomment-1722488870

`link` still contains `../..`. It is suggested to use [WHATWG URL API](https://nodejs.org/api/url.html#the-whatwg-url-api) instead of manipulating the url string.